modules/RTC/RTCUtils.js

```diff
@@ -1397,8 +1396,7 @@ class RTCUtils extends Listenable {
     isDeviceChangeAvailable(deviceType) {
         return deviceType === 'output' || deviceType === 'audiooutput'
             ? isAudioOutputDeviceChangeAvailable
-            : browser.isChromiumBased()
-                || browser.isFirefox() || browser.isEdge();
+            : browser.isChromiumBased() || browser.isFirefox();
```

Comment:
Doesn't this check cover all browsers now? If Safari is the exception we should be explicit about it IMHO.

Reply:
How about RN ? Is input device (both cam and mic) changing supported there ?

Reply:
No, but we don't support device preview anyway, it's switched as the user selects, so I don't think it's relevant.

Reply:
ok makes sense then, thanks for the explanation.

Reply:
I believe the reason Safari is an exception is because we have disabled video on Safari after it switched to Unified plan by default. Once we add unified plan support, we could remove that exception as well.

Reply:
Ah, gotcha! Good point! Let's keep it for now then.